tools/accept/t_w5_032.sh

```
@@ -0,0 +1,28 @@
+  # 直连既有 PostgreSQL（无 docker 的开发机/沙箱：自备 PG，DSN 指向管理员库）。
+  # 注意：0014 的 pii_vault_reader 是集群级角色，集群里若有其他库已 apply
+  # 0014（如测试用 muti_dev），down 全量的 DROP ROLE 会因跨库 ACL 依赖失败——
+  # 直连模式要求目标集群不被其他已迁移库共用
+  python tools/sql/migrate_check.py --admin-dsn "$MIGCHECK_ADMIN_DSN"
+else
+  # 默认路径：独立临时 PG16 实例（digest 与 compose 同锚定）
+  make migrate-go-check
+fi
+
+echo "== 验收 #5：migrate-go-check 已接入 make check（PR 阶段拦截）=="
+grep -q '\$(MAKE) migrate-go-check' Makefile || {
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)